test/helpers/StoreAndHeadersFactory.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace Wovnio\Test\Helpers;
+
+use Wovnio\Wovnphp\Store;
+use Wovnio\Wovnphp\Headers;
+
+class StoreAndHeadersFactory {
+  public static function fromFixture($fixture = 'default', $settingsOverwrite = array(), $envOverwrite = array()) {
+    $storeSettings = self::buildStoreOptions($settingsOverwrite);
+    $env = self::buildEnv($fixture, $envOverwrite);
+
+    return StoreAndHeadersFactory::get($env, $storeSettings);
+  }
+
+  public static function get($env, $settings = array()) {
+    $store = new Store($settings);
+    $headers = new Headers($env, $store);
+
+    return array($store, $headers);
+  }
+
+  private static function buildStoreOptions($options) {
+    $defaultOptions = array(
+      'default_lang' => 'en',
+      'supported_langs' => array('en'),
+      'url_pattern_name' => 'path',
+      'project_token' => 'KK9kZ'
```

Comment:
This is my User token :'( but I guess it's fine

Reply:
I took it from current tests, I'll use another one (hopefully there won't be a lot of tests to change).